resources/sync_gateway_configs/sync_gateway_sg_replicate_cc.json

```diff
@@ -9,12 +9,20 @@
     "log": ["CRUD+", "Cache+", "HTTP+", "Changes+", "Replicate"],
     "databases":{
         "db1":{
+            {{ autoimport }}
+            "unsupported": {
+                {{ xattrs }}
```

Comment:
The if is required if we have other unsupported flags in the config. The if will prepend xattrs to the existing unsupported flags with a comma.